app/src/main/java/com/belatrixsf/connect/managers/EmployeeManager.java

```diff
@@ -128,6 +132,22 @@ public void onFailure(ServiceError serviceError) {
         }
     }
 
+    private void registerDevice() {
+        if (PreferencesManager.get().getDeviceToken() != null) {
```

Comment:
store token in a variable, so you can reuse it if condition is met and void looking it up at the preferences.